incubator/command-dump/src/test/java/io/aklivity/zilla/runtime/command/dump/internal/airline/TsharkRunner.java

```diff
@@ -14,9 +14,7 @@
  */
 package io.aklivity.zilla.runtime.command.dump.internal.airline;
```

Comment:
Please move to `io.aklivity.zilla.runtime.command.dump.internal.test` package in test sources.